src/foremast/consts.py

```diff
@@ -154,6 +153,8 @@ def find_config():
                                  default='').split(','))
 DEFAULT_ELB_SECURITYGROUPS = set(validate_key_values(config, 'base', 'default_elb_securitygroups',
                                  default='').split(','))
+DEFAULT_SECURITYGROUP_RULES = json.loads(validate_key_values(config, 'base', 'default_securitygroup_rules',
```

Comment:
I think the variable should include something related that this is application level security group.